code/modules/projectiles/guns/energy/blaster.dm

```diff
@@ -28,6 +28,7 @@
 	max_shots = 12
 	origin_tech = list(TECH_COMBAT = 3, TECH_MAGNET = 2)
 	projectile_type = /obj/item/projectile/energy/blaster
+	slot_flags = SLOT_BELT
```

Comment:
shouldn't this be `slot_flags = SLOT_BELT|SLOT_HOSLTER`?

Reply:
The slot flags should actually be `null` here. In-game energy carbines do not fit in any belt or holster.